src/kustomer/webChat/README.md

```diff
@@ -0,0 +1,61 @@
+# Kustomer Integration Example
+
+This is a functioning service desk integration between Watson Assistant and Kustomer. 
+
+**Important:**  This is a reference implementation that provides an example of a fully functional integration. Make any necessary changes and perform robust testing before deploying this integration in production.
+
+This reference implementation supports the core features of a Kustomer integration. If you want to customize or extend it to add more features, follow the procedure described in the [README](../../../README.md) for this repository.
+
+## Overview
+
+This integration is based on the functionality of Kustomer Chat Core SDK, which allows client side to communicate with Kustomer Chat Platform. 
+
+The following Kustomer resources provide some useful guides on Kustomer custom chat implementation:
+  - [Build your own UI](https://developer.kustomer.com/chat-sdk/v2.0-Web/docs/build-your-own-ui)
+
+  - [Core API Reference](https://developer.kustomer.com/chat-sdk/v2.0-Web/docs/core-api-reference)
+
+  - [Authenticate chat](https://developer.kustomer.com/chat-sdk/v2.0-Web/docs/authenticate-chat-with-jwt-token)
+
+  - [Chat Management: Settings](https://kustomer.kustomer.help/en_us/chat-settings-B1sdKbtz7) 
+
+## Setting up
+
+1. If you haven't done so already, follow the setup steps in the root-level [README](../../../README.md#development) to make sure you can run an instance of [ExampleServiceDesk](../../example/webChat/README.md).
+
+2. Within the Kustomer Platform, generate a new API Key with the role `org.tracking`. Save the key, will need it for the next step.
+
+3. Edit the index.html in the top directory to include the follow script
+
+```
+<script src="https://cdn.kustomerapp.com/chat-web/core.js" data-kustomer-api-key="YOUR_KUSTOMER_API_KEY"></script>
+```
+
+4. To have the chat transcript display to an agent, within the Kustomer Platform create a new attribute on the Conversation Klass with Display Name as Watson Assistant Session History and set the type as Single Line Text 
```

Comment:
It would probably be good to add a little more info here about where you give find this in the UI. And you should probably add a note if this is *required* to get the integration working although I think with a tweak to the code, this doesn't have to be required.

Reply:
Will updated header from `Setting Up` to `Required setup`